src/reduced_basis/rb_construction_base.C

```diff
@@ -391,38 +381,33 @@ void RBConstructionBase<Base>::generate_training_parameters_random(const Paralle
   }
 
   // finally, set the values
-  {
-    std::map<std::string, std::unique_ptr<NumericVector<Number>>>::iterator it           = training_parameters_in.begin();
-    std::map<std::string, std::unique_ptr<NumericVector<Number>>>::const_iterator it_end = training_parameters_in.end();
+  for (auto & pr : training_parameters_in)
+    {
+      std::string param_name = pr.first;
```

Comment:
Okay, maybe these all belong in a separate PR...

Reply:
I'll just fix them now, I found only like one other one so far...